new.html

```diff
@@ -55,6 +58,60 @@
 
 		wickedElements.define('#grid', grid);
 
+
+		var alert = {
+			type: 'Warning',
+			message: 'You should check in on some of those fields below.',
+			dispatchEvent: function(event) {
+				if (this.el && typeof this.el.dispatchEvent == 'function') {
+					this.el.dispatchEvent(event);
+				}
+			},
+			init: function (event) {
+				console.log('init');
+				this.el = event.currentTarget;
+				this.render();
+			},
+			onconnected: function() {
+				console.log('connect');
+			},
+			onattributechanged: function(e) {
+				console.log('changed');
+				if (e.oldValue === e.newValue) {
+					return;
+				}
+				if (e.attributeName === 'type') {
+					this.type = e.newValue;
+					this.render();
+				} else if (e.attributeName === 'message') {
+					this.message = e.newValue;
+					this.render();
+				}
+			},
+			render: function() {
+				console.log('render');
+				this.el.innerHTML = '<strong alert-type>' + this.type + '</strong> <span class="message">' + this.message + '</span><button type="button" class="close" data-dismiss="alert"><span>&times;</span></button>';
+			}
+		};
+
+
+		wickedElements.define('.alert', alert);
+
+		var alertCount = 1;
+
+		function createAlert() {
```

Comment:
An example to create wicked elements dynamically which will still act as the same way if they were created before defining. In other words, we can create wicked elements after defining them as well.